akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/Behaviors.scala

```diff
@@ -183,24 +183,24 @@ object Behaviors {
    * wrapped behavior can evolve (i.e. return different behavior) without needing to be
    * wrapped in a `monitor` call again.
    */
-  def monitor[T](monitor: ActorRef[T], behavior: Behavior[T]): Behavior[T] =
-    scaladsl.Behaviors.monitor(monitor, behavior)
+  def monitor[T](interceptMessageClass: Class[_ <: T], monitor: ActorRef[T], behavior: Behavior[T]): Behavior[T] =
```

Comment:
I never remember, these type bounds work in Java because they become `<? extends T>`?

Reply:
It could have been useful for `monitor` as a filter for subclass of `T`, but I have removed it in 767311c22d8af483304f621532bd8072530b1179 since it seems it was mostly added by accident. If we would have such a feature it should probably be better as  predicate function.